internal/net/control/control_darwin.go

```diff
@@ -1,4 +1,5 @@
 //go:build darwin && !linux && !windows && !wasm && !js
+
```

Comment:
I'm not sure but is it okay to insert a blank line here?

Reply:
I think it is better not to insert blank line here... I will update the source code